tests/Rector/Architecture/Factory/NewObjectToFactoryCreateRector/config.yml

```diff
@@ -0,0 +1,3 @@
+services:
+  Rector\Rector\Architecture\Factory\NewObjectToFactoryCreateRector:
+    Rector\Tests\Rector\Architecture\Factory\NewObjectToFactoryCreateRector\Source\MyClass: []
```

Comment:
Please use 4-spacing as everywhere here. It shold be loaded `.editorconfig`. Do you have it installed in PHPStorm?

Reply:
No i did not, thanks for perfect tip 👍